docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -194,6 +152,12 @@ The data is now arranged as follows:
 
 ![Row-level geo-partitioning](/images/explore/multi-region-deployments/geo-partitioning-2.png)
 
+{{< tip title="Region-local transaction table" >}}
+
+When we create tables using a tablespace with a placement set, YugabyteDB automatically creates a transaction table under the tablespace if one does not yet exist, with a name like `system.transactions_90141438-f42c-4a39-8a12-4072c1216d46`.
```

Comment:
(I want to move all our docs to second-person, but at least let's get the notes that way :) )
```suggestion
When you create tables using a tablespace with a placement set, YugabyteDB automatically creates a transaction table under the tablespace if one doesn't yet exist, with a name like `system.transactions_90141438-f42c-4a39-8a12-4072c1216d46`.
```